src/Dashboard/Dashboard.js

```diff
@@ -2,29 +2,37 @@ import React, { Component } from 'react'
 import { Route, Switch, Link, Redirect } from 'react-router-dom'
 
 import Playdate from '../Playdate/Playdate'
-import './Dashboard.css'
+import './_Dashboard.scss'
 
 const Dashboard = ({ appointments, users }) => {
```

Comment:
I guess we'll have to see if this needs to be reconfigured as the back end for this has changed